src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
```

Comment:
This line of code is too long. Remember that lines of code should have 80 characters max. It is easier to read shorter lines.